ext/BijectorsTrackerExt.jl

```diff
@@ -529,7 +529,27 @@ end
 
 # NOTE: Probably doesn't work in complete generality.
 wrap_chainrules_output(x) = x
+function wrap_chainrules_output(x::ChainRulesCore.Thunk)
```

Comment:
Is this covered by the tests?

Reply:
No it isn't! _But_ do we want to? :confused: 
I just started writing a test, and the realized it means I have to move `wrap_chainrules__output` to Bijectors.jl itself rather than as an extension. _But_ it's only really used for Tracker (it is also used for one part in ReverseDiffjl, but this can be dropped in favor of the macro that has been added to ReverseDiff.jl).

Reply:
> No it isn't! 

But does that mean it's not needed?